packages/core/src/observable/types.ts

```diff
@@ -0,0 +1,26 @@
+/* Observable interop typing. Taken from https://github.com/ReactiveX/rxjs */
+
+/**
+ * An observable that aligns with the
+ * [TC39 observable proposal](https://github.com/tc39/proposal-observable) and
+ * can be consumed by other observable libraries like
+ * [rxjs](https://github.com/ReactiveX/rxjs) or
+ * [zen-observable](https://github.com/zenparsing/zen-observable).
+ */
+export interface Observable<T> {
+  subscribe(): Subscription
+  subscribe(observer: Partial<Observer<T>>): Subscription
+  subscribe(next: (value: T) => void): Subscription
```

Comment:
`onError? : Function, onComplete? : Function` are missing comparing to [TC39 observable proposal](https://github.com/tc39/proposal-observable)

Reply:
yeah i looked here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/zen-observable/index.d.ts#L36 insteadof here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/zen-observable/index.d.ts#L46

and i also had type errors with rxjs, because they introduce `null` on some props.

i will add this back in